misc/package/Data Files/MWSE/core/startLuaMods.lua

```diff
@@ -13,7 +13,7 @@ local function execLuaMod(runtime)
 		-- Setup our dependencies.
 		local dependencyManager = dependencyManagerModule.new({
 			metadata = runtime.metadata,
-			logger = logger.new({ name = name, logLevel = "INFO" }),
+			logger = logger.new{ modName = name, level = 3, modDir = runtime.key },
```

Comment:
It looks a bit weird to pass magic number 3. If info is the default log level can you just drop the `level` parameter?